locale/pt-br/texts.po

```diff
@@ -0,0 +1,1422 @@
+msgstr "Lembre-se de <strong>fazer o backup</strong> dessa configuração."
+
+#: src/screens/CreateToken.js:213
+msgid ""
+"Here you will create a new customized token. After the creation, you will be "
+"able to send this new token to other addresses."
+msgstr ""
+"Aqui você irá criar seu próprio token. Depois de criado, você conseguirá "
+"enviar seus novo token para outros endereços."
+
+#: src/screens/CreateToken.js:214
+msgid ""
+"Custom tokens share the address space with all other tokens, including HTR. "
+"This means that you can send and receive tokens using any valid address."
+msgstr ""
+"Os tokens compartilham os mesmos endereços, inclusive com o HTR. Isso "
+"significa que você pode enviar ou receber tokens usando qualquer endereço."
+
+#: src/screens/CreateToken.js:215
+msgid ""
+"Remember to make a backup of your new token's configuration string. You will "
+"need to send it to other people to allow them to use your new token."
+msgstr ""
+"Lembre-se de fazer o backup da configuração do seu novo token. Você irá "
+"precisar mandar essa configuração para que outras pessoas também consigam "
+"usar o seu token."
+
+#: src/components/ModalEditToken.js:86 src/screens/CreateToken.js:221
+msgid "Short name"
+msgstr "Nome curto"
+
+#: src/screens/CreateToken.js:222
+msgid "MyCoin"
+msgstr "MeuToken"
+
+#: src/components/ModalEditToken.js:89 src/screens/CreateToken.js:225
+msgid "Symbol"
+msgstr "Símbolo"
+
+#: src/screens/CreateToken.js:226
+msgid "MYC (max 5 characters)"
+msgstr "MEU (máximo 5 caracteres)"
+
+#: src/screens/CreateToken.js:231
+msgid "Amount"
+msgstr "Quantidade"
+
+#: src/screens/CreateToken.js:247
+msgid "Select address automatically"
+msgstr "Selecionar um endereço automaticamente"
+
+#: src/screens/CreateToken.js:252
+msgid "Destination address"
+msgstr "Endereço de destino"
+
+#: src/components/OutputsWrapper.js:48 src/components/tokens/TokenMint.js:112
+#: src/screens/CreateToken.js:253
+msgid "Address"
+msgstr "Endereço"
+
+#: src/screens/CreateToken.js:262
+#, javascript-format
+msgid "Token ${ _this.state.name } created"
+msgstr "O token ${ _this.state.name } foi criado"
+
+#: src/screens/CustomTokens.js:50
+msgid "Custom Tokens"
+msgstr "Tokens"
+
+#: src/screens/CustomTokens.js:51
+msgid ""
+"You can create your own digital token with customized specifications on "
+"Hathor Network with only a few clicks. They will fully work under the same "
+"technical assumptions of high scalability and decentralized consensus of our "
+"native HTR tokens. Custom tokens will always work independently of the price "
+"of the native HTR token, and they can serve multiple purposes."
+msgstr ""
+"Você pode criar seus próprios tokens digitais na Hathor Network com apenas "
+"alguns cliques. Eles irão possuir as mesmas características técnicas de alta "
+"escalabilidade e consenso descentralizado dos tokens nativos HTR. O seu "
+"token será independente do token nativo HTR e não haverá relação de preço "
```

Comment:
The translation is different from the original sentence here and "não haverá relação de preço entre eles" I still don't know if it's the best. Maybe just "Seu token criado pode ser utilizado para diversos objetivos e não dependerá do token nativo HTR ou do seu preço para operação." Indicating that after creating the token, you don't need to handle HTR anymore